app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -146,6 +141,28 @@ public void onTimelineChanged(Timeline timeline, int reason) {
         });
     }
 
+    /**
+     * Configures Exoplayer for video playback. Initially we try with core decoders, but allow
+     * ExoPlayer to silently fallback to software renderers.
+     *
+     * @param activity                   The associated activity
+     * @return A configured factory for Exoplayer
+     */
+    private DefaultRenderersFactory configureExoplayerFactory(Activity activity) {
```

Comment:
Receive the activity as context instead.

```suggestion
    private DefaultRenderersFactory configureExoplayerFactory(Context context) {
```